eng/scripts/Invoke-GeneratAndBuildV2.ps1

```diff
@@ -0,0 +1,87 @@
+#Requires -Version 7.0
+param (
+  [string]$inputJsonFile,
+  [string]$outputJsonFile
+)
+
+. (Join-Path $PSScriptRoot automation GenerateAndBuildLib.ps1)
+
+$inputJson = Get-Content $inputJsonFile | Out-String | ConvertFrom-Json
+$swaggerDir = $inputJson.specFolder
+$swaggerDir = Resolve-Path $swaggerDir
+$swaggerDir = $swaggerDir -replace "\\", "/"
+[string[]] $inputFilePaths = $inputJson.changedFiles;
+$readmeFiles = $inputJson.relatedReadmeMdFiles
+$commitid = $inputJson.headSha
+$repoHttpsUrl = $inputJson.repoHttpsUrl
+$downloadUrlPrefix = $inputJson.installInstructionInput.downloadUrlPrefix
+[string] $autorestConfig = $inputJson.autorestConfig
+
+$autorestConfigYaml = ""
+if ( $null -ne $autorestConfig -and $autorestConfig -ne "") {
```

Comment:
You can directly compare `if ($autorestConfigYaml)`, I guess. Or maybe `[string]::IsNullOrEmpty($autorestConfigYaml)`

Reply:
No. the script will support both scenarios : provide autorestConfig or not.

Reply:
updated